.travis.yml

```diff
@@ -4,12 +4,14 @@ before_install:
   - python --version
   - pip install -U pip
   - pip install -U pytest coverage pytest-cov
-  - pip install codecov black
+  - pip install codecov black mypy pylint
 
 install:
   - pip install -e . # install package
 
 script:
+  - pylint --rcfile=setup.cfg PeptideBuilder/
+  - mypy PeptideBuilder/
```

Comment:
For both the `pylint` and the `mypy` lines, could you add a brief comment stating what these lines do? Just like the next two lines, `black`, and `pytest`.